x64-uwp and arm-uwp failed due to this:
```
10>C:\agent\_work\1\s\buildtrees\boost-nowide-standalone\src\dalone_v10-98a85ebde3\test\test_system.cpp(85,38): error C2039: 'getenv': is not a member of 'std' [C:\agent\_work\1\s\buildtrees\boost-nowide-standalone\arm-uwp-dbg\test\nowide-test_system_n.vcxproj]
    16>C:\agent\_work\1\s\buildtrees\boost-nowide-standalone\src\dalone_v10-98a85ebde3\test\test_system.cpp(85,38): error C2039: 'getenv': is not a member of 'std' [C:\agent\_work\1\s\buildtrees\boost-nowide-standalone\arm-uwp-dbg\test\nowide-test_system_w.vcxproj]
```
arm64-windows failed due to this:
```
test_system.cpp.obj : error LNK2019: unresolved external symbol __imp_CommandLineToArgvW referenced in function "public: __cdecl nowide::args::wargv_ptr::wargv_ptr(void)" (??0wargv_ptr@args@nowide@@QEAA@XZ)
nowide-test_system_w.exe : fatal error LNK1120: 1 unresolved externals
```

--- a/ports/nowide/CONTROL
+++ b/ports/nowide/CONTROL
@@ -1,0 +1,4 @@
+Source: nowide
+Version: 10.0.1
+Homepage: https://github.com/boostorg/nowide
+Description: Boost nowide module (standalone)

--- a/ports/nowide/portfile.cmake
+++ b/ports/nowide/portfile.cmake
@@ -1,0 +1,24 @@
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/boostorg/nowide/releases/download/v10.0.1/nowide_standalone_v10.0.1.tar.gz"
+    FILENAME "nowide_standalone_v10.0.1.tar.gz"
+    SHA512 b349983127532fcfcb2bd29ce327634ea8d980e1da6a67fe44d0a5761a81c6cc78e518439970099b155732c3edb0fa8f1f1a1df5018d59b8cb699626c121f95e
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS -DBUILD_TESTING=OFF
+)
+
+vcpkg_install_cmake()
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/nowide TARGET_PATH share/${PORT})
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

